packages/cojson/src/tests/coValueCore.loadFromStorage.test.ts

```diff
@@ -0,0 +1,551 @@
+import { afterEach, beforeEach, describe, expect, test, vi } from "vitest";
+import { CoValueCore, idforHeader } from "../coValueCore/coValueCore";
+import { CoValueHeader } from "../coValueCore/verifiedState";
+import { RawCoID } from "../ids";
+import { LocalNode } from "../localNode";
+import { StorageAPI } from "../storage/types";
+import {
+  createTestMetricReader,
+  createTestNode,
+  tearDownTestMetricReader,
+} from "./testUtils";
+
+let metricReader: ReturnType<typeof createTestMetricReader>;
+
+beforeEach(() => {
+  metricReader = createTestMetricReader();
+});
+
+afterEach(() => {
+  tearDownTestMetricReader();
+});
+
+function setup() {
```

Comment:
This could live in `testUtils`. It could be something like `createCoValueCoreWithoutContent`, that returns the covalue and the header so that you can mark it as available later